authz/audit/stdout_logger.go

```diff
@@ -0,0 +1,64 @@
+/*
+ *
+ * Copyright 2023 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package audit
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type StdOutLogger struct{}
+
+func (logger *StdOutLogger) Log(event *Event) {
+	jsonBytes, err := json.Marshal(event)
```

Comment:
This is not sufficient. First of all we need custom marshaling for things like `FullMethodName` if we want `rpc_method` instead. Secondly, we need the timestamp.

Reply:
Agreed for the timestamp but what's the usecase when `FullMethodName` would be marshaled incorrectly? I tried with a dummy `/helloworld.Greeter/SayHello` value and it looks fine

Reply:
Done